core/events/services/classic/models.py

```diff
@@ -290,9 +290,9 @@ def update_from_submission(self, submission: domain.Submission) -> None:
         self.submitter_id = submission.creator.native_id
         self.submitter_name = submission.creator.name
         self.submitter_email = submission.creator.email
-        self.is_author = int(submission.submitter_is_author)
-        self.agree_policy = int(submission.submitter_accepts_policy)
-        self.userinfo = int(submission.submitter_contact_verified)
+        self.is_author = 1 if submission.submitter_is_author else 0
+        self.agree_policy = 1 if submission.submitter_accepts_policy else 0
+        self.userinfo = 1 if submission.submitter_contact_verified else 0 
```

Comment:
What is the behavior that we capture with this construction vs. `int()` casts? Seems like `None` is the major issue - `None` is `False`-y, but `int(None)` returns a `TypeError`. Does `None` occur "in the wild" with these fields now?

Reply:
Yes, that's the primary reason. We do permit ``None``, since ``False`` alone does not sufficiently differentiate between the case that the author has asserted a ``False`` value and the case that the author has made no assertion.